README.md

```diff
@@ -1119,14 +1089,14 @@ import (
 )
 
 func main() {
-	bytes, _ := hex.DecodeString("72656769737465726564")
-	fmt.Println(string(bytes))
+	bytes, _ := base64.StdEncoding.DecodeString("cmVnaXN0ZXJlZA==")
+    fmt.Println(string(bytes))
```

Comment:
Not needed any longer, use standard `base64` command (it's a part of coreutils, so it's widely available).

Reply:
I can not find it, could you, please, give a full method name?

Reply:
No methods, pure CLI.
```
$ echo ... | base64 -d
```
Way easier than this helper in Go.